lib/templates/deploy.bash.ruby.template

```diff
@@ -9,11 +15,25 @@ initializeDeploymentConfig() {
 		OPTIONS="--without $BUNDLE_WITHOUT";
 		echo "Bundle install with options $OPTIONS";
 	fi
+	
+	if [ -z $BUNDLE_INSTALL_LOCATION ]; then 
```

Comment:
We can prepend "RUBY". I document all env variables in the README on ruby github